Assets/Mirror/Runtime/NetworkIdentity.cs

```diff
@@ -93,12 +93,13 @@ public sealed class NetworkIdentity : MonoBehaviour
         /// A unique identifier for NetworkIdentity objects within a scene.
         /// <para>This is used for spawning scene objects on clients.</para>
         /// </summary>
-        public ulong sceneId => m_SceneId;
+        // persistent scene id <sceneHash/32,sceneId/32> (see AssignSceneID comments)
+        public ulong sceneId;
```

Comment:
add a FormerlySerializedAs for the old one so existing projects don't get all new scene ids

Reply:
Done